MonopolyKata/Monopoly.cs

```diff
@@ -76,14 +84,44 @@ private void DetermineOrder()
             }
         }
 
-        public void RunMonopoly()
+        public void RunMonopoly(Int32 numberOfRounds)
         {
             var i = 0;
-            while (i < 20)
+            while (i < numberOfRounds)
             {
                 var j = 1;
-                while (j < 5){
-                    order[j].RollDicePair();
+                while (j < 5)
+                {
+                    //var MoneyTransfer = 0;
+                    //roll dice
+                    order[j].RollDicePair(gameBoard);
+                    var currentLocation = order[j].currentLocation;
+                    order[j].BasicAccountTransfers(currentLocation, gameBoard);
+                    //buy properties
+                    switch (gameBoard.GetStatus(currentLocation))
+                    {
+                        case "UNAVAILABLE":
+                            if (gameBoard.GetOwnerName(currentLocation) != order[j].ToString())
+                            {
+                                var owner = DetermineOwnerOfLocation(currentLocation);
+                                var type = gameBoard.GetType(currentLocation);
+                                var multiplier = CalculateMultipleGroupMultiplier(owner, currentLocation);
+                                var ownerOfProperty = stringToPlayer[owner];
+                                var rentOwed = gameBoard.GetInitialRent(currentLocation) * multiplier;
+
+                                order[j].accountBalance -= rentOwed;
+                                ownerOfProperty.accountBalance += rentOwed;
+                            }
+                            break;
+                        case "AVAILABLE":
+                            order[j].PurchaseProperties(currentLocation, gameBoard);
+                            gameBoard.SetOwnerName(currentLocation, order[j].ToString());
+                            break;
+                        case "LOCKED":
```

Comment:
Look into using an enum rather than magic strings to designate the status of a space.